### DISCUSSION TITLE
How to get the logic gate count?

### DISCUSSION THREAD

**Discussion by ghost** [hneemann/Digital#758] [Q&A] [answered]
When designing my circus, i need to know how many logic gates my circus use so that i can fit it into a fpga board. However, i can't find this option in the menu. Does Digital have this feature?

**Comment by hneemann** [comment, marked as answer]
Digital offers a stats function via _Simulation->Circuit Statistics_.

But if the circuit fits into a certain FPGA depends very much on the FPGA. How large the LUT's are, how many adders, multipliers, etc. are available on the chip, how much distributed RAM is available, etc.

This can only be calculated by the analysis tool of the FPGA manufacturer.
